src/views/Transfer.vue

```diff
@@ -90,8 +90,11 @@ export default {
       this.$refs.transferSnackbar.open()
     },
     onClose (type) {
+      if (!this.$store.getters.checkForActiveNode) {
```

Comment:
It needs for avoid sending transactions when nodes were disabled in settings

Reply:
That's wrong. Nodes way become active/inactive any moment. Instead of doing these useless checks just make sure that ADM API errors are caught and handled correctly.